config/manager/usde-config.yaml

```diff
@@ -47,11 +52,21 @@ data:
       includeSubfields: true
     - path: spec.vertices.udf.builtin
       includeSubfields: true
-    - path: spec.sideInputs
+    - path: spec.sideInputs.container.image
+    - path: spec.sideInputs.container.command
+    - path: spec.sideInputs.container.args
+    - path: spec.sideInputs.container.env
       includeSubfields: true
-    - path: spec.templates
+    - path: spec.sideInputs.container.envfrom
+      includeSubfields: true
```

Comment:
broke all of this out into the individual fields

Reply:
Great idea!